string[] ids
geometry_msgs/Polygon[] polygons
